requirements/pip.txt

```diff
@@ -18,9 +18,12 @@ django-vanilla-views==1.0.4
 pytest-django==2.8.0
 
 requests==2.3.0
-slumber==0.6.0
+slumber==0.6
```

Comment:
Yeah, I'm investigating why the test fails on py2 [here](https://github.com/rtfd/readthedocs.org/pull/1793) things works for me locally, so I'm trying to narrow down which version make test fails.